### PR TITLE
Add OTP Auth to the list of authenticator apps

### PR DESCRIPTION
**Pull request by atm0915**
This adds OTP Auth to the list of authenticator apps, solving issue #496

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Tested with the following apps:
 * [Google Authenticator](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2) (proprietary)
 * [KeePassXC (Linux, Windows, macOS)](https://keepassxc.org/) (open-source) Available via [download](https://keepassxc.org/download/), package repositories or [GitHub](http://www.github.com/keepassxreboot/keepassxc/) (Keepass also provides a plugin and Keepass2Android allows to use TOTP token)
 * [SailOTP (SailfishOS)](https://github.com/seiichiro0185/sailotp) (open source) Available via JollaStore or [Openrepos.net](https://openrepos.net/content/seiichiro0185/sailotp)
-
+* [OTP Auth](https://cooperrs.de/otpauth.html) (proprietary) Availabe via [Apple's App Store](https://itunes.apple.com/us/app/otp-auth/id659877384)
```
Comment (ChristophWurst) [inline README.md]: could you please move the line up by one so that there is no blank line to the other list items but one below?

Comment (atm0915) [reply README.md]: There is no blank line as seen in this screen shot
image

Comment (ChristophWurst) [reply README.md]: Take a look at the source. It does render correctly, but looks weird there.

**Comment by atm0915** [comment]
I had to create a new fork for some reason so I created a new pull request, #515. Sorry about that.

**Pull request closed without merging**